packages/@aws-cdk/aws-apigatewayv2/test/http/route.test.ts

```diff
@@ -174,6 +175,18 @@ describe('HttpRoute', () => {
           connectionType: HttpConnectionType.VPC_LINK,
           uri: 'some-target-arn',
           secureServerName: 'some-server-name',
+          parameterMapping: new ParameterMapping(
```

Comment:
Can you add a separate unit test for parameter mapping instead?

Reply:
Done